### ISSUE TITLE
Add option for Custom AI functionality using URL of OpenAI compatible API

### ISSUE THREAD

**Issue by cmgriffing** [gitkraken/vscode-gitlens#3223]
I would like to be able to set a custom URL for an OpenAI compatible API. This would be useful for fine tuned custom LLMs for specific projects. It would also be really useful in an Enterprise environment that wants to prevent exfiltrating data outside of their network.

**Comment by cmgriffing-at-gitkraken** [comment]
@SpaceMerlin33 Could you take a look at this Issue?

**Comment by chestm007** [comment]
I think this issue can be closed, as this appears implemented?
<img width="883" height="1012" alt="Image" src="https://github.com/user-attachments/assets/faa2443e-5323-4bb3-abaa-0d016af9a9db" />

**Comment by cmgriffing** [comment]
Yes, sorry. I kept this around for launchpad use when making videos. But we should get rid of this now that I have more repos to use as examples.
